src/main/external-resources/renderers/Sony-PlayStation3.conf

```diff
@@ -39,14 +39,15 @@ Supported = f:mp4        v:mp4|h264               a:aac                m:video/m
 Supported = f:wmv        v:wmv|vc1                a:wma                m:video/x-ms-wmv           n:2
 
 # Supported audio formats:
+Supported = f:aac     a:(?!alac).+   n:2             m:audio/x-m4a
+Supported = f:atrac                  n:2             m:audio/x-oma
```

Comment:
@Nadahar At the beginning you asked for review and now wait...???
Come on, i'm in so i will try to finish it.